deployments/polygon/usdc/configuration.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Compound USDC (Polygon)",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x2791bca1f2de4661ed88a30c99a7a9449aa84174",
+  "baseTokenPriceFeed": "0xfE4A8cc5b5B2366C1B58Bea3858e81843581b2F7",
+  "borrowMin": "100e6",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0e15",
+    "baseBorrowSpeed": "0e15",
+    "baseMinForRewards": "1000000e6"
+  },
+  "assets": {
```

Comment:
tossed in some assets (with some arbitrary `borrowCF`, `liquidateCF` and `liquidationFactor` values); additions/removals/adjustments welcome